### ISSUE TITLE
Notes get "eaten" when using Internal Trilium Link function

### ISSUE THREAD

**Issue by johnson0915** [zadam/trilium#959]
Hi there! 
I just discovered the app and I love it! Thanks you and the others for developing it!
However, it seems that with the newest version of the application (I downloaded "trilium-windows-x64-0.41.1") notes will be "eaten," i.e., transformed into the name of the page to which the notes are meant to be linked. 

For example, here's the original page that contain the notes that I took during a lecture. The selected part is the portion of the notes that I'd like to link to another page called "summary". 
Before: 
![image](https://user-images.githubusercontent.com/26791011/79031934-d6fc5180-7ba2-11ea-9e0e-0c23bebfeb00.png)

When I make use of the function "Internal Trilium Link," what happens is that this part gets "eaten" or transformed into the name of the summary page. 
After
![image](https://user-images.githubusercontent.com/26791011/79031956-04e19600-7ba3-11ea-9d80-878ab1af72c9.png)

Could someone please tell me how to solve this problem or fix the bug? 

Thanks for reading and time!
Stay safe!

**Comment by zadam** [comment]
Hello, thank you for reporting this bug, this has been broken from 0.41 which introduced a different type of link.

This has been fixed, as a workaround in the meantime you can use `CTRL-L` without selected text, there you can define both link and it's title manually.

**Comment by johnson0915** [comment]
Hi zadam!

Thanks for solving this bug! And no worries! It's my pleasure to be able to
contribute to the improvement of Trilium!

Again, thanks very much for developing this great app!
Stay safe!

Best regards,
BangGao Jin

zadam <notifications@github.com> 于2020年4月11日周六 上午9:49写道：

**Comment by johnson0915** [comment]
Ah btw, I was wondering if you had some  plans for developing the features
that I mentioned in " https://github.com/zadam/trilium/issues/960  [feature
request] Making linked notes "viewable" in the root page and improving LINK
MAP possibly."

Basically these days I've been looking for a note-taking app that I can use
to implement the Zettelkasten Method, and I've tried almost all options on
the market (roam research, OneNote, Evernote,  TiddlyWiki, Zettlr, and many
many more). However, it seems like that only your application--Trilium--and
Roam Research support both the feature of bidirectional link and the
feature of "making-whatever-part-of-the-text-linked-to-another-page." And
since I'd like to settle on an app that I can use for the next 20/30/40
years(I'd like to be a scholar one day so longevity of the app and fully
control of my notes are definitely among my top considerations), Trilium is
much much better than Roam, for the latter is a cloud-based service, which
means that all my notes and "relationships" that I've created can go away
at any time. This is something that I really really dislike or even hate.
And I think having the features that I mentioned would makem Trilium very
suitable and valuable for those who're interested in using the Zettelkasten
Method like me, which means that there would be more users.

Therefore, might I know if you have some intention of adding those features?

Thanks for reading!

Again, at any rate, thanks for making Trilium! It's a fantastic open-source
app!

Best regards,
BangGao Jin
